net-libs/grpc/grpc-1.12.1.ebuild

```diff
@@ -0,0 +1,154 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python2_7 python3_{4,5,6} )
+DISTUTILS_OPTIONAL=1
+
+inherit distutils-r1 flag-o-matic toolchain-funcs
+
+PROTOBUF_VERSION="3.5.2"
```

Comment:
Add a small note saying to check this when bumping the package and exactly where to find this version in the release so we dont forget.

Reply:
I just looked at submodule version. Will add.